### PR TITLE
feat: build and publish kuma-net-ebpf docker image

### PR DESCRIPTION
**Pull request by bartsmykla**
Instead of including logic to work with merbridge/ebpf docker images we should do it inside kuma-net, so this PR introduces the logic and configuration for github actions to build and publish `kumahq/kuma-net-ebpf` docker image

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits): 👍 

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?: 👍

### CODE REVIEW

File: tools/builds/ebpf/compile.mk [hunk 1]
```diff
@@ -0,0 +1,58 @@
+CC=clang
+CFLAGS=-O2 -g  -Wall -target bpf -I/usr/include/$(shell uname -m)-linux-gnu
+
+MACROS :=
+DEBUG ?=
+
+MESH_MODE ?= istio
```
Comment (slonka) [inline tools/builds/ebpf/compile.mk:7]: istio?

File: tools/builds/ebpf/load-and-attach.mk [hunk 2]
```diff
@@ -0,0 +1,118 @@
+PROG_MOUNT_PATH ?= /sys/fs/bpf
+TC_PINNING_PATH ?= $(PROG_MOUNT_PATH)/tc/globals
+
+CGROUP2_PATH ?= $(shell mount | grep cgroup2 | awk '{print $$3}' | grep -v "^/host" | head -n 1)
+ifeq ($(CGROUP2_PATH),)
+$(error It looks like your system does not have cgroupv2 enabled, or the automatic recognition fails. Please enable cgroupv2, or specify the path of cgroupv2 manually via CGROUP2_PATH parameter.)
+endif
+
+# Map
+load-map-cookie_original_dst:
```
Comment (slonka) [inline tools/builds/ebpf/load-and-attach.mk:10]: where are these targets used?

**Review by slonka — APPROVED** [review]
Looks alright, just 2 questions. Did you already run this? I don't see images here: https://hub.docker.com/u/kumahq

File: tools/builds/ebpf/compile.mk (hunk 1 above)
Comment (bartsmykla) [reply tools/builds/ebpf/compile.mk:7]: It's a little bit stripped down version of merbridge's Makefile, so I didn't change anything except removing unnecessary targets

Comment (slonka) [reply tools/builds/ebpf/compile.mk:7]: Can you link to the original one in a comment?

Comment (slonka) [reply tools/builds/ebpf/compile.mk:7]: Can you link to the original one in a comment?

File: tools/builds/ebpf/load-and-attach.mk (hunk 2 above)
Comment (bartsmykla) [reply tools/builds/ebpf/load-and-attach.mk:10]: https://github.com/kumahq/kuma-net/blob/master/ebpf/ebpf.go#L121-L159

**Comment by bartsmykla** [comment]
@slonka PR had to be merged before I would be able to run this, and I'm on it now

**Pull request merged**